site/category.xml

```diff
@@ -80,7 +79,9 @@
 	</feature>
 
 <!-- foundation -->
-
+	<feature id="org.jboss.tools.foundation.license.feature" version="0.0.0">
+		<category name="AllFeatures"/>

```

Comment:
No, we don't need one.
license features just like as templates for other features, they are not installed so the source feature is useless.